wiki/Beatmap_Spotlights/Seasons/2023_Autumn/en.md

```diff
@@ -0,0 +1,94 @@
+---
+tags:
+  - charts
+  - Ranking Charts
+  - Seasonal Spotlights
+---
+
+# Beatmap Spotlights Season 10 (Autumn 2023)
+
+## Schedule
+
+| Event | Time |
+| :-- | :-- |
+| Season start | Oct 23, 2023 |
+| Playlist lobby: Playlist A | Oct 23 - Nov 6, 2023 |
+| Playlist lobby: Playlist B | Nov 6 - Nov 20, 2023 |
+| Playlist lobby: Playlist C | Nov 20 - Dec 4, 2023 |
+| Playlist lobby: Playlist A | Dec 4 - Dec 11, 2023 |
+| Playlist lobby: Playlist B | Dec 11 - Dec 18, 2023 |
+| Playlist lobby: Playlist C | Dec 18 - Dec 25, 2023 |
+| Season conclusion | Dec 25, 2023 |
+
+## Weekly multiplayer lobbies
+
+As of now, joining the weekly multiplayer lobbies requires participants to download and install [osu!(lazer)](/wiki/Client/Release_stream/Lazer).
```

Comment:
```suggestion
Joining the weekly multiplayer lobbies requires participants to download and install [osu!(lazer)](/wiki/Client/Release_stream/Lazer).
```
`As of now` feels pretty unnecessary because we've been doing it for a long while and aren't planning on stopping anytime soon